extensions/benchmark/README.md

```diff
@@ -3,320 +3,4 @@
-
-#### Remote location
-
-Use `--model-url` option to load a model from a URL. The URL must point to an archive file.
-
-The following is a pytorch model
-
-```
--u https://alpha-djl-demos.s3.amazonaws.com/model/djl-blockrunner/pytorch_resnet18.zip
-```
-We would recommend to make model files in a zip for better file tracking.
-
-#### Local directory
-
-Use `--model-path` option to load model from a local directory or an archive file.
-
-Mac/Linux
-
-```
--p /home/ubuntu/models/pytorch_resnet18
-or
--p /home/ubuntu/models/pytorch_resnet18.zip
-```
-
-Windows
-
-```
--p C:\models\pytorch_resnet18
-or
--p C:\models\pytorch_resnet18.zip
-```
-
-If the model file name is different from the parent folder name (or the archive file name), you need
-to specify `--model-name` in the `--args`:
-
-```
--n traced_resnet18
-```
-
-### Step 3: Define how many runs you would like to make
-
-add `-c` inside with a number
-
-```
--c 1000
-```
-
-This will run 1000 times inference.
-
-### Step 4: Define your model inputs
-
-The benchmark script uses dummy NDArray inputs.
-It will make fake NDArrays (like `NDArray.ones`) to feed in the model for inference.
-
-If we would like to fake an image:
-
-```
--s 1,3,224,224
-```
-
-This will create a NDArray (DataType FLOAT32) of shape(1, 3, 224, 224).
-
-If your model requires multiple inputs like three NDArrays with shape 1, 384 and 384. You can do the followings:
-
-```
--s (1),(384),(384)
-```
-
-If you input `DataType` is not FLOAT32, you can specify the data type with suffix:
-
-- f: FLOAT32, this is default and is optional
-- s: FLOAT16 (short float)
-- d: FLOAT64 (double)
-- u: UINT8 (unsigned byte)
-- b: INT8 (byte)
-- i: INT32 (int)
-- l: INT64 (long)
-- B: BOOLEAN (boolean)
-
-For example:
-
-```
--s (1)i,(384)f,(384)
-```
-
-### Optional Step: multithreading inference
-
-You can also do multi-threading inference with DJL. For example, if you would like to run the inference with 10 threads:
-
-```
--t 10
-```
-
-Best thread number for your system: The same number of cores your system have or double of the total cores.
-
-You can also add `-l` to simulate the increment load for your inference server. It will add threads with the delay of time.
-
-```
--t 10 -l 100
-```
-
-The above code will create 10 threads with the wait time of 100ms.
-
-## Advanced use cases
-
-For different purposes, we designed different mode you can play with. Such as the following arg:
-
-```
--d 86400
-```
-
-This will ask the benchmark script repeatedly running the designed task for 86400 seconds (24 hour).
-If you would like to make sure DJL is stable in the long run, you can do that.
-
-You can also keep monitoring the DJL memory usages by enable the following flag:
-
-```
-export BENCHMARK_OPTS="-Dcollect-memory=true"
-```
-
-The memory report will be made available in `build/memory.log`.
+**This module has been moved to [deepjavalibrary/djl-serving/benchmark](https://github.com/deepjavalibrary/djl-serving/tree/master/benchmark).**
```

Comment:
I left the sentence "dll-bench is a command line tool that makes it easy for you to benchmark the model on different
platforms.". I also added the section about different comparisons you can make with the tool. I didn't want to grab the engine list because it will go out of date at some point. Does this seem fine?